OpenFastPath/ofp
# Update to odp v1.35.0.0 level

- Handle deprecations
- Fix a bunch of gcc-10 and other warnings
- Add missing term calls to tests

v2:
- Add commit `Update ODP repository links` (Matias)
- Add commit `timer: Use odp_timer_pool_param_init() instead of memset()` (Matias)
- Add commit `readme: Remove references to travis-ci`
- Add commit `ipsec: If op mode is async, check whether scheduled queues are supported` (Janne)
- Add commit `Remove ThunderX specific documentation`

v3:
- Add review tags.
